core/build.gradle.kts

```diff
@@ -25,6 +25,7 @@ dependencies {
     api(libs.kotlin.stdlib)
     implementation(libs.retrofit.core)
     implementation(libs.retrofit.converter.gson)
+    implementation(libs.google.gson)
```

Comment:
Sadly due to our bundling set up, you'll have to add this to gcpbuildcache/build.gradle.kts and s3buildcache/build.gradle.kts as these transitive dependencies don't get included.